.kitchen.yml

```diff
@@ -1,19 +0,0 @@
----
```

Comment:
That is interesting.  What part will fail?  Would kitchen-ec2 also be cut off from use?